src/main/docs/guide/introduction/whatsNew.adoc

```diff
@@ -127,12 +127,17 @@ With the new `micronaut-kubernetes-client` module you can now inject apis object
 
 In Micronaut 3 this new module will be used as primary kubernetes client, making the current one deprecated.
 
+=== Micronaut AWS
```

Comment:
This should be nested under "Cloud Features"

Reply:
Done